### PR TITLE
Add PAI environment setup steps in DOC

### PR DESCRIPTION
**Pull request by SparkSnail**

### CODE REVIEW

File: docs/en_US/TrainingService/PaiMode.md [hunk 1]
```diff
@@ -3,7 +3,33 @@
 NNI supports running an experiment on [OpenPAI](https://github.com/Microsoft/pai) (aka pai), called pai mode. Before starting to use NNI pai mode, you should have an account to access an [OpenPAI](https://github.com/Microsoft/pai) cluster. See [here](https://github.com/Microsoft/pai#how-to-deploy) if you don't have any OpenPAI account and want to deploy an OpenPAI cluster. In pai mode, your trial program will run in pai's container created by Docker.
 
 ## Setup environment
-Install NNI, follow the install guide [here](../Tutorial/QuickStart.md).
+Step 1. Install NNI, follow the install guide [here](../Tutorial/QuickStart.md).
```
Comment (xuehui1991) [inline docs/en_US/TrainingService/PaiMode.md]: do we need to point out that the setup steps is for paiyarn or paiK8s?

Comment (SparkSnail) [reply docs/en_US/TrainingService/PaiMode.md]: No, this doc is only for paik8s, do not need to point out again. paiYarn has another doc `paiYarnMode.md`.

File: docs/en_US/TrainingService/PaiMode.md [hunk 2]
```diff
@@ -3,7 +3,33 @@
...
+![](../../img/pai_data_management_page.jpg)  
+The `DEFAULT_STORAGE`field is the path to be mounted in PAI's container when a job is started. The `Preview container paths` is the NFS host and path that PAI provided, you need to mount the corresponding host and path to your local machine first, then NNI could use the PAI's NFS storage.  
+For example, use the following command:
+```
+sudo mount nfs://gcr-openpai-infra02:/pai/data /local/mnt
+```
+Then the `/data` folder in container will be mounted to `/local/mnt` folder in your local machine.  
+You could use the following configuration in your NNI's config file:
+```
+  nniManagerNFSMountPath: /local/mnt
```
Comment (ultmaster) [inline docs/en_US/TrainingService/PaiMode.md]: Extra indent?

File: docs/en_US/TrainingService/PaiMode.md [hunk 3]
```diff
@@ -3,7 +3,33 @@
 NNI supports running an experiment on [OpenPAI](https://github.com/Microsoft/pai) (aka pai), called pai mode. Before starting to use NNI pai mode, you should have an account to access an [OpenPAI](https://github.com/Microsoft/pai) cluster. See [here](https://github.com/Microsoft/pai#how-to-deploy) if you don't have any OpenPAI account and want to deploy an OpenPAI cluster. In pai mode, your trial program will run in pai's container created by Docker.
 
 ## Setup environment
-Install NNI, follow the install guide [here](../Tutorial/QuickStart.md).
+Step 1. Install NNI, follow the install guide [here](../Tutorial/QuickStart.md).  
+Step 2. Get PAI token.
```
Comment (ultmaster) [inline docs/en_US/TrainingService/PaiMode.md]: Suggest adding an extra blank line between steps.

File: docs/en_US/TrainingService/PaiMode.md (hunk 2 above)
Comment (SparkSnail) [reply docs/en_US/TrainingService/PaiMode.md]: fixed.

File: docs/en_US/TrainingService/PaiMode.md (hunk 3 above)
Comment (SparkSnail) [reply docs/en_US/TrainingService/PaiMode.md]: fixed.

**Pull request merged**